tests/test_experiment.py

```diff
@@ -611,3 +620,128 @@ def test_experiment_stop_does_not_raise_on_unknown_job_id(
     assert stat == InvalidJobStatus.NEVER_STARTED
     after_cancel = exp.get_status(*all_known_ids)
     assert before_cancel == after_cancel
+
+
+def test_start_sequences_of_jobs_good(test_dir, wlmutils, monkeypatch, job_list):
+    """Test the unpacking of a tuple of tuples
+    exp.start(job1, (job2, job3))
```

Comment:
This test might need to be changed, looks like rn you start a sequence of Jobs but you might need to add some logic to directly test the work you added! This could be removing Ensemble and App inits as well as Experiment init and writing a unit test instead of integration, this would allow you to test the nitty gritty.  Instead you could use unittest.mock.patch to assert that _helpers was called during start by calling assert_called_once() on the mock object, I dont think you need to check the return here. Checking the return might fit better in a test_helper directory and write a test for the _helper where you pass in MockJobs and assert the return is flattened! Your test below where you change the sequence pattern would work really well here in unit test form.